data/json/monsters.json

```diff
@@ -210,7 +210,7 @@
     "vision_day":40,
     "hp":16,
     "death_function":"NORMAL",
-    "description":"A spotted wild cat living across much of North America.  It is not a serious threat to humans, but it can be aggressive if not left alone.",
+    "description":"A spotted wild cat, these live across much of North America.  It is not a serious threat to humans, but it can be aggressive if not left alone.",
```

Comment:
The original phrase may imply the specific bobcat the character encounters has been going abroad across North America, which does not make sense. One of my goals is to reduce confusion; so, I will revise the trickier part.

I will make a new commit tomorrow with a simpler phrase on bobcat entry.
I will also transition more of the phrases to sentences in the monsters.json file.

Thanks for the quick turnaround.